plugins/defaultplugins/ifplugin/model/interfaces/interfaces.proto

```diff
@@ -19,6 +28,7 @@ message Interfaces {
         bool enabled = 4;
         string phys_address = 5;
         uint32 mtu = 6;
+        RxModeType rx_mode = 7;
```

Comment:
add support for queues. Maybe you can turn RxModeType field to message like this:

message RxMode {
      RxModeType rx_mode
      QueueID uint32
      QueueIDValid uint32
}